src/lib/DutchDecayLib.sol

```diff
@@ -28,7 +28,7 @@ library DutchDecayLib {
         view
         returns (uint256 decayedAmount)
     {
-        if (decayEndTime < decayStartTime) {
+        if (decayEndTime <= decayStartTime) {
```

Comment:
we really wanna revert if theyre the same? is it because they shouldnt be using a decay and should just have a limit order?

Reply:
https://github.com/Uniswap/UniswapX/pull/194

I think it was just undefined behavior what we should do in that case

Reply:
https://github.com/Uniswap/UniswapX/blob/main/audit/v1.1/OpenZeppelin.pdf L-03